src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax;
+
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class RegistrationServiceTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user = new User();
```

Comment:
You should initialize the `User` object within each test method or in a setup method annotated with `@BeforeEach` to ensure test isolation and prevent side effects between tests.